lang/test/org/partiql/lang/eval/builtins/functions/MathFunctionsTest.kt

```diff
@@ -0,0 +1,35 @@
+package org.partiql.lang.eval.builtins.functions
+
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.ArgumentsSource
+import org.partiql.lang.eval.EvaluatorTestBase
+import org.partiql.lang.eval.builtins.ExprFunctionTestCase
+import org.partiql.lang.util.ArgumentsProviderBase
+
+class MathFunctionsTest : EvaluatorTestBase() {
+
+    @ParameterizedTest
+    @ArgumentsSource(MathFunctionsPassCases::class)
+    fun runPassTests(tc: ExprFunctionTestCase) = runEvaluatorTestCase(
+        tc.source,
+        expectedResult = tc.expectedLegacyModeResult
+    )
+
+    class MathFunctionsPassCases : ArgumentsProviderBase() {
+
+        override fun getParameters(): List<Any> = listOf(
+            ExprFunctionTestCase("ceil(1)", "1"),
+            ExprFunctionTestCase("ceil(1.0)", "1"),
+            ExprFunctionTestCase("ceil(1.1)", "2"),
+            ExprFunctionTestCase("ceil(-42.8)", "-42"),
+            ExprFunctionTestCase("ceiling(1)", "1"),
+            ExprFunctionTestCase("ceiling(1.0)", "1"),
+            ExprFunctionTestCase("ceiling(1.1)", "2"),
+            ExprFunctionTestCase("ceiling(-42.8)", "-42"),
+            ExprFunctionTestCase("floor(1)", "1"),
+            ExprFunctionTestCase("floor(1.0)", "1"),
+            ExprFunctionTestCase("floor(1.1)", "1"),
+            ExprFunctionTestCase("floor(-42.8)", "-43"),
+        )
+    }
```

Comment:
Can also add some error/negative test cases. Consider using the `checkInvalidArgType` and `checkInvalidArity` APIs that Ziqi had created. Example using those APIs for the `size` ExprFunction: https://github.com/partiql/partiql-lang-kotlin/blob/ecbaebc70b66d437d5ebed53e31b4c4e8315aeca/lang/test/org/partiql/lang/eval/builtins/functions/SizeEvaluationTest.kt#L42-L57

Reply:
Yeah I was curious about the negative tests, because they don't necessarily test the function -- more like testing the logic for how function signatures are checked/enforced. I can add some negative tests as well.